### PR TITLE
Adding Markdown file to replace the current txt

### PR DESCRIPTION
**Pull request by loumalouomega**
Just a modification removing txt and replaced by a Markdown file for aesthetic enhancement

### CODE REVIEW

**Comment by pooyan-dadvand** [comment]
I'm not agree with changing the third party library files in our repository. And even less for aesthetic. It adds unnecessary administration overhead.

**Comment by loumalouomega** [comment]
The only change is a readme, which it was originally added by me, I just change the format to Markdown. The only question is if I can use the logo, if not I can just removed

File: applications/MeshingApplication/custom_external_libraries/mmg/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+[//]: <> (![http://www.mmgtools.org/](https://www.mmgtools.org/files/2016/09/logo_mmg.png  "http://www.mmgtools.org/") TODO: Ask permision to use the image)
```
Comment (loumalouomega) [inline applications/MeshingApplication/custom_external_libraries/mmg/README.md]: Question, here, can I use it?

**Comment by pooyan-dadvand** [comment]
Now I see. if it is a readme about how to use it in kratos put it in the app of kratos but without logo as I'm not sure neither if we can put it. put it as kratos part I mean.

**Review by loumalouomega — COMMENTED** [review]
OK, I will remove the logo

**Comment by loumalouomega** [comment]
Can anyone approve this?

**Review by RiccardoRossi — APPROVED** [review]
:+1:

**Pull request merged**